docs/GettingStartedDocs/VisualStudioWindows.md

```diff
@@ -10,6 +10,7 @@ Studio Code, see the
 You will need the following:
 
 - A development machine with support for SGX1 or SGX1 with Flexible Launch Control (FLC) (see [instructions for determining the SGX support level](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) if needed).
+  - If you would like to install the Open Enclave Host-Verify SDK and build evidence verification applications without enclaves, then this is not necessary. For more information about the Open Enclave Host-Verify SDK, please look [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_host_verify_Windows.md).
```

Comment:
```suggestion
  - If you would like to install the Open Enclave Host-Verify SDK and build evidence verification applications without enclaves, then this is not necessary. For more information about the Open Enclave Host-Verify SDK, see the [installation instructions](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_host_verify_Windows.md).
```
(avoid "here" as a link label)

Reply:
Ah that's a good one. I saw it on some old documents and followed suit. I'll adopt your suggestion, thanks!